### PR TITLE
[FEATURE] changes to e2e using ngsiv2

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

**Comment by XavierVal** [comment]
A little bit of info about in the description could be helpful

File: iotqatools/cb_v2_utils.py [hunk 1]
```diff
@@ -1687,3 +1698,9 @@ def get_subscription_context(self):
         :return dict (see "constructor" method by dict fields)
         """
         return self.subscription_context
+
+    def get_request_response_string(self):
+        """
+        return a string with request and another one to response
+        """
+        return self.request_string, self.response_string
```
Comment (XavierVal) [inline iotqatools/cb_v2_utils.py]: No new line at end of file :)

**Comment by XavierVal** [comment]
LGTM

File: iotqatools/cb_v2_utils.py (hunk 1 above)
Comment (iariasleon) [inline iotqatools/cb_v2_utils.py]: Fixed in c5e4826

**Pull request merged**